test/quantization/core/test_quantized_op.py

```diff
@@ -4678,6 +4678,184 @@ def test_qlinear_add_relu_pt2e(self):
         qlinear = torch.ops.onednn.qlinear_pointwise.binary
         self._test_qlinear_pt2e_helper(qlinear, "add_relu")
 
+    def _quantize_fp8e4m3(self, t: torch.Tensor, channelwise: bool, scale: Optional[torch.Tensor] = None):
+        quant_max = torch.finfo(torch.float8_e4m3fn).max
+        eps = torch.Tensor([torch.finfo(torch.float32).eps])
+        if channelwise:
+            scale = scale or t.reshape(t.shape[0], -1).abs().max(-1)[0] / quant_max
```

Comment:
Should it be `t.reshape(-1, t.shape[-1])`?

Reply:
If weight shape = [OC, K1, K2, ...], here we reshape it to [OC, K]. Then we find the abs max value along K. So, we get OC by `t.shape[0]` and reshape the tensor to `[t.shape[0], -1]`